extensions/wikia/EditPageLayout/js/editor/WikiaEditor.js

```diff
@@ -1097,11 +1106,13 @@
 		},
 
 		proxyAllCommandsState: function() {
+			
 			for (var commandName in this.stateProxiedCommands)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Restrict what this loop acts on by testing each property. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AForIn)

[project-id]: wikia:app